internal/handlers/common/params.go

```diff
@@ -258,3 +258,61 @@ func addNumbers(v1, v2 any) (any, error) {
 		return nil, errUnexpectedLeftOpType
 	}
 }
+
+// GetPositiveNumber returns doc's value for key or protocol error for missing or invalid parameter.
+func GetPositiveNumber(document *types.Document, key string) (int32, error) {
```

Comment:
Let's use `GetWholeNumberParam` in that function

Reply:
Fixed